core/trino-main/src/main/java/io/trino/metadata/DisabledSystemSecurityMetadata.java

```diff
@@ -195,6 +195,9 @@ public Optional<Identity> getFunctionRunAsIdentity(Session session, CatalogSchem
         return Optional.empty();
```

Comment:
Please improve a commit message and pull request title and description. It is more about propagating events of function creation than handling onwnership.